clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -108,13 +112,21 @@ private enum SubscriptionType {
     public synchronized String toString() {
         return "SubscriptionState{" +
             "type=" + subscriptionType +
-            ", subscribedPattern=" + subscribedPattern +
+            ", subscribedPattern=" + subscribedPatternInUse() +
```

Comment:
I'm intentionally avoiding differentiating the 2 patterns output thinking that it will help have a smooth transition when we end up deprecating/removing the old pattern in the future. Makes sense?